src/app/modules/submission/submission.utils.ts

```diff
@@ -195,6 +195,15 @@ export const isAttachmentResponse = (
   )
 }
 
+/**
+ * Checks if a response is a quarantined attachment response to be processed by the virus scanner.
+ */
+export const isQuarantinedAttachmentResponse = (
```

Comment:
Because for the new flow with the quarantine key, content will not have been sent by the FE, so there will not be content at the point of time this check is used.